core/src/main/java/com/opensymphony/xwork2/interceptor/ParametersInterceptor.java

```diff
@@ -18,20 +18,20 @@
  */
 package com.opensymphony.xwork2.interceptor;
 
-import com.opensymphony.xwork2.ActionContext;
-import com.opensymphony.xwork2.ActionInvocation;
-import com.opensymphony.xwork2.TextProvider;
-import com.opensymphony.xwork2.XWorkConstants;
+import com.opensymphony.xwork2.*;
```

Comment:
better import explicit classes like in L25-29?

Reply:
Strange... IDEA should use `*` in case of importing more than 5 classes from the same package, fixed!